src/components/EditDeleteBar/EditDeleteBar.test.jsx

```diff
@@ -0,0 +1,88 @@
+import { fireEvent, queryByRole, render, screen } from '@testing-library/react';
+import EditDeleteBar from './EditDeleteBar';
+
+describe('<EditDeleteBar/>', () => {
+  it('should render the edit and remove buttons when given a remove logic', () => {
+    const isVisible = true;
+    const toggleEditing = jest.fn();
+    const handleRemove = jest.fn();
+
+    const { getByRole } = render(
+      <EditDeleteBar
+        isVisible={isVisible}
+        toggleEditing={toggleEditing}
+        handleRemove={handleRemove}
+      />);
```

Comment:
You can use a function to abstract behavior that's repeated across a test file like this. Functionally, it's the same but makes it easier to add/edit/read tests in my opinion. Something like:
```typescript
const setupComponent = (isVisible, toggleEditing, handleRemove) => {
    return (
        render(
            <EditDeleteBar
                isVisible={isVisible}
                toggleEditing={toggleEditing}
                handleRemove={handleRemove}
            />
        )
    )
}
```

Reply:
Generally I abstain from using generic functions in tests because, as commits go by, they might end up having complex logic (and who tests a test? 😂) and they get prone to hiding important details of the story a test "tells". But in this case I agree it's very reasonable as long as the function stays simple!

Reply:
Yes, I didn't go much into detail because you'll see them in our codebase, but we use them to abstract particularly common logic across related tests. A few examples of the logic we put there:
* Get values for the props as params
* Add default values for props when they're not informed in the params
* Get feature flags values
* Get context values (e.g. current user, locale, company...)